db/migrate/20220218083013_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.string :comments_counter
```

Comment:
- [ ] `comments_counter` should be of type `integer` instead of `string`. Please update it.